app/src/main/java/org/schulcloud/mobile/data/DataManager.java

```diff
@@ -97,12 +98,18 @@ public String getAccessToken() {
                     }
                 });
     }
+    public void signOut()
+    {
```

Comment:
Please take care of proper formatting. No new line for opening blocks with curly brackets in Java.